testnet/stacks-node/src/node.rs

```diff
@@ -345,6 +345,15 @@ impl Node {
         }
 
         let burnchain_config = config.get_burnchain();
+
+        // instantiate DBs
+        let _burnchain_db = BurnchainDB::connect(
+            &burnchain_config.get_burnchaindb_path(),
+            &burnchain_config,
+            true,
+        )
+        .expect("FATAL: failed to connect to burnchain DB");
```

Comment:
What required adding this?

Reply:
The cherry-picked StackerDB patches that I brought in from `master` which, as mentioned in the comment, were required in order to make this PR's tests work.  Without these cherry-picked patches, the nodes would never talk to each other and would instead ban one another due to a bug in `next`'s current StackerDB implementation.